bindings/node/lib/autoEncrypter.js

```diff
@@ -126,34 +126,39 @@ module.exports = function (modules) {
         mongoCryptOptions.logger = options.logger;
       }
 
-      if (options.extraOptions && options.extraOptions.csflePath) {
-        mongoCryptOptions.csflePath = options.extraOptions.csflePath;
+      if (options.extraOptions && options.extraOptions.cryptSharedLibPath) {
+        mongoCryptOptions.cryptSharedLibPath = options.extraOptions.cryptSharedLibPath;
```

Comment:
Thank you for addressing this as well! :blue_heart:

@nbbeeken @durran @dariakp @baileympearson Heads up that this is a breaking change from 2.1.0 – that should be okay on our side, and I don’t think the team really announced shared library support, so I’d be okay with not introducing aliases here.

I’ve opened https://github.com/mongodb/node-mongodb-native/pull/3280 to adjust the typings.